mesonbuild/minstall.py

```diff
@@ -76,7 +76,9 @@ def __exit__(self, exception_type, value, traceback):
             append_to_log(self.lf, d)
 
 def is_executable(path, follow_symlinks=False):
-    '''Checks whether any of the "x" bits are set in the source file mode.'''
+    '''
+    Here checks whether any of the "x" bits are set in the source file mode.
```

Comment:
Do not add "Here" to the front; in fact it should be imperative tense, see [PEP257](https://www.python.org/dev/peps/pep-0257/).

Reply:
Fixed.

Reply:
Thanks for the reference.